### PR TITLE
Shatterline: Now running on proton

### PR DESCRIPTION
**Pull request by Kreyren**
ref: https://github.com/ValveSoftware/Proton/issues/6165#issuecomment-1258731897

### CODE REVIEW

**Comment by Curve** [comment]
Please leave the anti cheat as is, we don't differentiate eos and eac.

Also i guess this superseeds #777 ?

File: games.json [hunk 1]
```diff
@@ -4557,13 +4557,17 @@
...
         "native": false,
-        "reference": "https://www.protondb.com/app/2087030",
+        "reference": "https://github.com/ValveSoftware/Proton/issues/6165#issuecomment-1258731897",
         "anticheats": [
-            "Easy Anti-Cheat"
+            "Epic Online Services"
+        ],
+        "notes": [
+            [
+                "Requires windows release set on 7 (e.g. 'protontricks win7') to workaround the use of Winsock Input/Output (RIO) API which is not implemented in wine/proton"
```
Comment (Curve) [inline games.json:4568]: Make this shorter, in-depth Details are mostly uninteresting for most people

**Comment by Kreyren** [comment]
> Please leave the anti cheat as is, we don't differentiate eos and eac. -- @Curve (https://github.com/Starz0r/AreWeAntiCheatYet/pull/821#issuecomment-1260749378)

You should, they are not the same.

> Also i guess this supersedes #777 ? -- @Curve (https://github.com/Starz0r/AreWeAntiCheatYet/pull/821#issuecomment-1260749378)

Seems like it.. the anti-cheat was broken for ~4 days after update due to a pipeline bug on upstream.

**Comment by Curve** [comment]
> > Please leave the anti cheat as is, we don't differentiate eos and eac. -- @Curve (https://github.com/Starz0r/AreWeAntiCheatYet/pull/821#issuecomment-1260749378)
> 
> You should, they are not the same.
> 
 While they're not exactly the same, they're still most recognizable as Easy Anti-Cheat - I don't think differentiating here is worth it.

**Comment by Kreyren** [comment]
> While they're not exactly the same, they're still most recognizable as Easy Anti-Cheat - I don't think differentiating here is worth it. -- @Curve (https://github.com/Starz0r/AreWeAntiCheatYet/pull/821#issuecomment-1260754294)

It is worth it, if upstream was using EAC the issue would be way different.. like mixing apples and oranges

File: games.json (hunk 1 above)
Comment (Kreyren) [reply games.json:4568]: I won't the details are important, merge as is or make your own version

Comment (Curve) [reply games.json:4568]: > I won't the details are important, merge as is or make your own version

You can add more in-depth details in the Updates section by linking to the relevant post that contains the information.  

Thanks for the PR, will adjust this on my own then

**Pull request closed without merging**